### ISSUE TITLE
About the effect of batch normalization

### ISSUE THREAD

**Issue by yxqlwl** [ClementPinard/FlowNetTorch#8]
Hi,
       It seems that FlowNetSBN performs worse than FlowNetS with batch_size = 8. Is it because the batch_size is not large enough? Thanks!

**Comment by ClementPinard** [comment]
Hello thanks for your observation.
Didn't try 8 batchSize but you may be right, as batchNormalization is doing a normalization over the whole batch. So the more data you have, the closer it will be to "running_std and "running_mean" which are the linear parameters when infering data.

Unfortunately i don't have time anymore to do a full training on this code. But if you have nice graphs showing better progress, feel free to do a PR !

**Comment by yxqlwl** [comment]
Thanks. I will try to add some graphs. :)
